src/classes/PS_IntegrationServiceConfig.cls

```diff
@@ -66,12 +90,86 @@ public with sharing class PS_IntegrationServiceConfig {
     * @return Boolean
     */
     public Boolean hasIntegrationPermissions() {
```

Comment:
Should this method be called isIntegrationUser? There is only one integration user (the user that last registered Elevate). However, there could be multiple users allowed to integrate with elevate (any user that has the correct perm set/s assigned to it). This method will be used to answer the former not the latter.